Source/Core/Core/PowerPC/Interpreter/Interpreter_LoadStorePaired.cpp

```diff
@@ -124,11 +120,6 @@ float Interpreter::Helper_Dequantize(const u32 _Addr, const EQuantizeType _quant
 	case QUANTIZE_S16:
 		fResult = static_cast<float>((s16)Memory::Read_U16(_Addr)) * m_dequantizeTable[_uScale];
 		break;
-
-	default:
```

Comment:
I can only assume laziness with the thought that a default will catch invalid types.